core/metrics/metrics_test.go

```diff
@@ -2,12 +2,13 @@ package metrics_test
 
 import (
 	"context"
-	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/alecthomas/assert"
```

Comment:
what's the point of this?

Reply:
if i edit the file, undo the changes, then save, gopls still reorders the imports

Reply:
btw I meant using 	"github.com/alecthomas/assert" instead of 	"github.com/stretchr/testify/assert"

Reply:
Wrong import inference I think, wasn't intentional. Removed now